lib/Backends/Backends.cpp

```diff
@@ -15,13 +15,6 @@
  */
 
 #include "Interpreter/Interpreter.h"
```

Comment:
Can we remove the #include of Interpreter.h as well?

Reply:
Good catch! I missed it initially. Removed now.